spec/requests/portfolios_spec.rb

```diff
@@ -173,14 +173,28 @@
   describe 'DELETE admin tagged /portfolios/:portfolio_id' do
     let(:valid_attributes) { { :name => 'PatchPortfolio', :description => 'description for patched portfolio' } }
 
-    context 'when :portfolio_id is valid' do
+    context 'when discarding a portfolio' do
       before do
         delete "#{api}/portfolios/#{portfolio_id}", :headers => admin_headers, :params => valid_attributes
       end
 
       it 'deletes the record' do
         expect(response).to have_http_status(204)
       end
+
+      it 'sets the discarded_at column' do
+        expect(Portfolio.with_discarded.find_by(:id => portfolio_id).discarded_at).to_not be_nil
+      end
+
+      it "can't be requested" do
```

Comment:
I'd like to avoid contractions: `cannot`

Reply:
I'd like that too 😉

Reply:
* in tests.

Reply:
Noted. Changing it now.